From ab69ba27d5767c93fa4c2ad5cf18008841810de4 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 15 Jan 2017 20:21:10 +0100
Subject: [PATCH 6/7] Replace SGML comment with an actual reference to the
 policy process

This gets rid of a comment, so we do not have to bother with restoring
it when converting to DocBook.
---
 policy.sgml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/policy.sgml b/policy.sgml
index fb1f30f..0fb3067 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -222,7 +222,8 @@
 	  id="mailto:debian-policy@lists.debian.org">. Proposals
           are discussed there and inserted into policy after a certain
           consensus is established.
-          <!-- insert shameless policy-process plug here eventually -->
+          The current policy process is described in the <url name="Process"
+          id="Process.md"> document.
           The actual editing is done by a group of maintainers that have
           no editorial powers. These are the current maintainers:
 
-- 
2.11.0

